client/main.js

```diff
@@ -0,0 +1,136 @@
+function httpGetAsync(theUrl, callback, message)
+{
+    var xmlHttp = new XMLHttpRequest();
+    xmlHttp.onreadystatechange = function() { 
+        if (xmlHttp.readyState == 4 && xmlHttp.status == 200)
+            callback(xmlHttp.responseText);
+    }
+    xmlHttp.open("POST", theUrl, true); // true for asynchronous 
+    xmlHttp.setRequestHeader("Content-Type", "application/json");
+    xmlHttp.send(JSON.stringify({ mfcc: message }));
+}
+
+function callback(res) {
+    console.log(res);
+}
+function doSetTimeout(a, i) {
+    setTimeout(function() {
+        var features = null;
+        features = a.get([
+            'mfcc'
+        ]);
+        console.log(features.mfcc)
+        httpGetAsync('http://127.0.0.1:80', callback, features.mfcc);
+    }, 500*i);
```

Comment:
From the paper, it says 30-millisec window size and 10 millisecond frame shift
I guess this should be 10 * i?